### ISSUE TITLE
 Gran Turismo 2 on PSX bettle HW Vulkan Glitchy RetroArch General

### ISSUE THREAD

**Issue by HighBunkerHxH** [libretro/beetle-psx-libretro#272]
Hello, I´ve downloaded Retroarch on Windows and I tested some PSone games on Bettle PSX HW with Vulkan, most of them worked pretty great but Gran Turismo 2 is pretty unplayable, in game menus is perfect but in in-game race, the HUD and the cars always Flickers and it runs very choppy, on the Bettle PSX Software it doesn´t have this problem and it runs perfectly, is there a fix that I can do?

    OS - Windows 10 64-bit Creators Update
    GPU - Nvidia GTX 750 ti
    CPU - I5 4690K 3,50 Ghz

**Comment by hizzlekizzle** [comment]
No, it is indeed a problematic game. The only solution is to run it with the software renderer.

**Comment by HighBunkerHxH** [comment]
oh bummer, it´s funny because both GT1 and GT2 use the same engine and I thought GT2 would run just as good as GT1
